community/libpkcs11-helper/sources

```diff
@@ -0,0 +1 @@
+https://github.com/OpenSC/pkcs11-helper/archive/pkcs11-helper-1.25.1.tar.gz
```

Comment:
Try this source: `https://github.com/OpenSC/pkcs11-helper/releases/download/pkcs11-helper-1.25.1/pkcs11-helper-1.25.1.tar.bz2`. It should remove the autoreconf requirement.